lib/logstash/filters/grok/timeout_enforcer.rb

```diff
@@ -0,0 +1,77 @@
+class LogStash::Filters::Grok::TimeoutEnforcer
```

Comment:
Let's wait till we have a second use case. Things that can properly use `InterruptedException` are rare, and I don't want to make more use now for something that we may need only in the future.